app/src/main/java/com/onelist/service/ItemService.kt

```diff
@@ -8,17 +8,15 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
-interface IItemService {
-    //TODO: Create the Interface for the Item Service
-
-    //TODO: Create the method to get all the items
-}
+interface IItemService {}
 
 class ItemService : IItemService {
     suspend fun fetchItems(): List<Item>? {
         return withContext(Dispatchers.IO) {
             val service = RetrofitClientInstance.retrofitInstance?.create(IItemDAO::class.java)
-            val items = async {service?.getAllItems()}
+            val items = async {
+                service?.getAllItems()
+            }
```

Comment:
Formatting can be performed automatically, and consistently, by the IDE (auto format code, organize imports).  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.